docs/integrations/sources/facebook-marketing.md

```diff
@@ -202,6 +202,7 @@ The Facebook Marketing connector uses the `lookback_window` parameter to repeate
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                                                                                                                                                                                           |
 |:--------|:-----------|:---------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 3.0.1   | 2024-06-01 | [38845](https://github.com/airbytehq/airbyte/pull/38845) | Update AdsInsights fields - removed   `cost_per_conversion_lead` and `conversion_lead_rate`                                                                                                                                                           |
```

Comment:
nit: `3.1.0`, since this is a breaking change on the Airbyte side.

Reply:
We should also update the `facebook-marketing-migrations.md` filed to be able to notify Customers with the breaking changes:

path: `.../docs/integrations/sources/facebook-marketing-migrations.md`

Please add the following line after the headline:
```
## Upgrading to 3.1.0

The `AdsInsights` Reports now don't have the possibility to fetch the next root level properties (fields):
- cost_per_conversion_lead
- conversion_lead_rate

### Refresh affected AdsInsights Report and clear data:

1. Select **Connections** in the main navbar.
   1. Select the connection(s) affected by the update.
2. Select the **Schema** tab. 
    1. Select **Refresh source schema**. 
    2. Select **OK** 
    3. Select **Save changes** at the bottom of the page.
   :::note
   Any detected schema changes will be listed for your review.
   :::

3. Navigate to a connection's **Settings** tab and click **Clear data** to clear all streams. This action will clear data for all streams in the connection. To clear data for a single stream navigate to the **Status** tab,  click the **three grey dots** next to the affected stream, and select **Clear data**. Do this for all affected streams in the connection.

For more information on clearing your data in Airbyte, see [this page](/operator-guides/clear).
```

Reply:
Done